dspace-oai/src/main/java/org/dspace/xoai/app/XOAIItemCompilePlugin.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.xoai.app;
+
+import com.lyncode.xoai.dataprovider.xml.xoai.Metadata;
+import org.dspace.content.Item;
+import org.dspace.core.Context;
+
+
+/**
+ * Used to enrich item.compile field description
+ *
+ */
+public interface XOAIItemCompilePlugin {
```

Comment:
As referred this name is confusing to me. Perhaps with a more detailed description I can help you finding a better name.

Reply:
done